site/gatsby-site/src/components/discover/REFINEMENT_LISTS.js

```diff
@@ -128,4 +128,14 @@ const REFINEMENT_LISTS = [
   },
 ];
 
-export default REFINEMENT_LISTS;
+const FIRST_ROW = REFINEMENT_LISTS.filter((refinement) =>
+  [
+    'classifications',
+    'epoch_incident_date',
+    'epoch_date_published',
+    'source_domain',
+    'language',
+  ].includes(refinement.attribute)
+);
```

Comment:
I think we should slice the first 5 instead, in case we end up updating the refinements or change their order

Reply:
Good point. I also changed the order so that the first row would show the ones I would guess are most interesting, in the absence of data.

![image](https://github.com/responsible-ai-collaborative/aiid/assets/25443411/a85f4624-5963-4740-b808-1f43c63f411f)

I also remembered that I changed the wording on the expand/collapse filters button, so those need translations. I went with "More filters" = "Ver más filtros" = "Plus de filtres" // "Fewer filters" = "Ver menos filtros" = "Moins de filtres". Let me know if those are incorrect or not idiomatic.

Test coming up...